server.ts

```diff
@@ -140,11 +142,72 @@ createConnection(ormOptions)
 			// get a user repository to perform operations with user
 			const userRepository = getManager().getRepository(User);
 
+			const existingUser = await userRepository.find({
+				where: {
+					id: req.params.id,
+				},
+			});
+			console.log("exisstingUser:", existingUser);
+			if (existingUser && existingUser.length == 0) {
+				res.send("User is not existing");
+				return;
+			}
+
 			userRepository.delete(req.params.id);
 			res.end();
 			return;
 		});
 
+		// update user data
+		app.put("/:id", async function (req, res) {
+			const userRepository = getManager().getRepository(User);
+
+			const existingUser = await userRepository.find({
+				where: {
+					id: req.params.id,
+				},
+			});
+			if (existingUser && existingUser.length == 0) {
+				res.send("User is not existing");
+				return;
+			}
+
+			const user = {
+				email: req.body.email,
+				name: req.body.name,
+				password: req.body.password,
+				profession: req.body.profession,
+			};
+
+			const updatedUser = await userRepository.update(req.params.id, user);
+			res.send(updatedUser);
+		});
+
+		// partial update user data
+		app.patch("/:id", async function (req, res) {
+			const userRepository = getManager().getRepository(User);
+
+			const existingUser = await userRepository.find({
+				where: {
+					id: req.params.id,
+				},
+			});
+			if (existingUser && existingUser.length == 0) {
+				res.send("User is not existing");
+				return;
+			}
+			console.log(req.body);
+			console.log(existingUser);
+
+			const updateData = req.body;
+
+			const updatedUser = await userRepository.update(
```

Comment:
I want to update and then return changed values, could you please help me ?

For example:
updateData
```
{
"name" : "asaka san"
}
```
```
Output 
{
"id": xx,
"name" : "asaka san"
}
```